### PR TITLE
Fix creating and editing articles with content_moderation uninstalled

### PR DESCRIPTION
**Pull request by alexpott**
Make sure these boxes are checked before submitting your pull request - thank you!

- [x] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [x] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/develop/docs/development.md#how-to-run-the-tests))
- [x] Necessary update hooks are provided.
- [x] User roles have correct access for new introduced permission.
- [x] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [x] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: tests/src/FunctionalJavascript/ArticleCreationTest.php [hunk 1]
```diff
@@ -119,6 +119,27 @@ public function testCreateArticle() {
       ->elementsCount('xpath', '//div[contains(@class, "field--name-field-paragraphs")]/div[contains(@class, "field__item")][9]//span[contains(@data-pin-id, "478085316687452268")]', 2);
   }
 
+  /**
+   * Test Creation of Article without content moderation.
+   */
+  public function testCreateArticleWithNoModeration() {
+    // Delete all the articles so we can disable content moderation.
+    foreach (\Drupal::entityTypeManager()->getStorage('node')->loadByProperties(['type' => 'article']) as $node) {
```
Comment (chrfritsch) [inline tests/src/FunctionalJavascript/ArticleCreationTest.php:127]: Interesting. Locally I had to remove all the moderation states in order to be able to disable the content_moderation module. The content could stay for me.

Comment (alexpott) [reply tests/src/FunctionalJavascript/ArticleCreationTest.php:127]: Yeah that would work... I just did it one way that came to mind.

Comment (chrfritsch) [reply tests/src/FunctionalJavascript/ArticleCreationTest.php:127]: Ok funny. The uninstall form recommends deleting the moderation states, but removing the content also works.
Anyway...

**Review by chrfritsch — APPROVED** [review]
This looks good to me

**Pull request merged**